src/parser/monk/brewmaster/modules/Abilities.ts

```diff
@@ -56,6 +56,30 @@ class Abilities extends CoreAbilities {
           static: 1000,
         },
       },
+      {
+        spell: SPELLS.CHI_BURST_TALENT,
+        category: Abilities.SPELL_CATEGORIES.ROTATIONAL,
+        cooldown: 30,
+        enabled: combatant.hasTalent(SPELLS.CHI_BURST_TALENT.id),
+        castEfficiency: {
+          suggestion: true,
+        },
+        gcd: {
+          base: 1500,
+        },
+      },
+      {
+        spell: SPELLS.CHI_WAVE_TALENT,
+        category: Abilities.SPELL_CATEGORIES.ROTATIONAL,
+        cooldown: 15,
+        enabled: combatant.hasTalent(SPELLS.CHI_WAVE_TALENT.id),
+        castEfficiency: {
+          suggestion: true,
+        },
+        gcd: {
+          base: 1500,
```

Comment:
chi wave has a 1s hasted GCD AFAIK

Reply:
I copied the values from MW/WW. Do you know if it should be the same for these specs?

Reply:
I just finished some basic testing in-game. Seems like it's 1.5 for MW, and 1.0 for BM/WW. Good catch!